### DISCUSSION TITLE
部署在云服务器上，10000端口无法访问页面。

### DISCUSSION THREAD

**Discussion by ElliotChina** [illacloud/illa-builder#997] [Q&A] [answered]
请问大神们，如何解决？使用的root用户安装部署的。
docker日志有报错，不知道是不是这个问题导致的。

<img width="1084" alt="image" src="https://user-images.githubusercontent.com/21995807/220583368-7918ddd4-fdcc-40d0-8557-f45152ea2741.png">

**Comment by ElliotChina** [comment, marked as answer]
看起来像没有权限，但是是用root用户直接部署的，也尝试了加入docker用户组，错误依然存在。

**Reply by karminski** [reply]
@ElliotChina 您好，您的操作系统是什么型号？然后docker是如何安装的呢？我们在我们本地模拟您的环境看一下

**Reply by ElliotChina** [reply]
使用的是腾讯云的轻应用服务器，系统版本CentOS 7.9。
docker好像是腾讯云自带的宝塔面板自动安装的，版本是Docker CE 23.0.1。
谢谢🙏

**Reply by karminski** [reply]
@ElliotChina 您好, CentOS 7 不在我们的支持范围内. 请使用 linux 5.4 以上内核的操作系统. 例如 Ubuntu 20.04
